app/models/patient.rb

```diff
@@ -49,7 +56,31 @@ def latest_scheduled_appointment
   end
 
   def latest_blood_pressure
-    latest_blood_pressures.first
+    blood_pressures.order(device_created_at: :desc).first
```

Comment:
`latest_blood_pressures` is used along with includes to eager load blood pressures. This will not be helpful in those cases no?

Reply:
I'm resolving this since it's unrelated to this PR.